spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/TaskControllerTests.java

```diff
@@ -151,6 +157,59 @@ public void testSave() throws Exception {
 	}
 
 	@Test
+	public void testCompose() throws Exception {
+		assertEquals(0, repository.count());
+		TaskDefinition testTask = new TaskDefinition("task", "task");
```

Comment:
Moreover, I understand this is not testing the parser etc, but a non-trivial composition would be welcome and would make it more clear what the test does